analyzers/src/SonarAnalyzer.CSharp/Rules/DoNotUseOutRefParameters.cs

```diff
@@ -56,19 +55,18 @@ protected override void Initialize(SonarAnalysisContext context)
 
                     var parameterSymbol = c.SemanticModel.GetDeclaredSymbol(parameter);
 
-                    if (!(parameterSymbol?.ContainingSymbol is IMethodSymbol containingMethod) ||
-                        containingMethod.IsOverride ||
-                        !containingMethod.IsPubliclyAccessible() ||
-                        IsTryPattern(containingMethod, modifier) ||
-                        containingMethod.GetInterfaceMember() != null)
+                    if (!(parameterSymbol.ContainingSymbol is IMethodSymbol { } containingMethod)
```

Comment:
I would not remove the null-coallescence from here, the symbol can be missing if a library was not restored yet and it would be great to avoid AD0001 errors in this case.
```suggestion
                    if (parameterSymbol?.ContainingSymbol is not IMethodSymbol containingMethod
```

Reply:
I think this wasn't addressed.